cachito/web/utils.py

```diff
@@ -4,7 +4,6 @@
 
 from flask import request, url_for
 
-
```

Comment:
And this one as well could be better to move into https://github.com/release-engineering/cachito/pull/564/commits/b491af65ae02c577309ef13a7e497f1dccd2289a